test/e2e/tests/port-stream-chunking/port-stream-chunking.spec.ts

```diff
@@ -0,0 +1,87 @@
+import assert from 'node:assert';
+import { Mockttp } from 'mockttp';
+import { Browser } from 'selenium-webdriver';
+import { getEventPayloads, withFixtures } from '../../helpers';
+import FixtureBuilder from '../../fixture-builder';
+import { loginWithBalanceValidation } from '../../page-objects/flows/login.flow';
+import HomePage from '../../page-objects/pages/home/homepage';
+import { MOCK_META_METRICS_ID } from '../../constants';
+
+const isFirefox = process.env.SELENIUM_BROWSER === Browser.FIREFOX;
+
+async function mockSegment(mockServer: Mockttp) {
+  return [
+    await mockServer
+      .forPost('https://api.segment.io/v1/batch')
+      .withJsonBodyIncluding({
+        batch: [{ type: 'track', event: 'Port Stream Chunked' }],
+      })
+      .thenCallback(() => {
+        return {
+          statusCode: 200,
+        };
+      }),
+  ];
+}
+
+describe('Port Stream Chunking', function () {
+  it('can load the wallet UI with a huge background state (~128MB)', async function () {
+    // add MOCK_TRANSACTION_BY_TYPE.HUGE to an array a bunch of times
+    const hugeTx = {
+      id: 4243712234858512,
+      time: 1589314601567,
+      status: 'confirmed',
+      chainId: '0x5',
+      loadingDefaults: false,
+      txParams: {
+        from: '0xabca64466f257793eaa52fcfff5066894b76a149',
```

Comment:
just as nit, if we used the fixture account here, as well as the chainId 1337, we could be able to verify this is actually present in the Activity tab (just adds an extra assert, but not essential for the test)-- however, I changed that and I still cannot see it 🤔 just wondering why might be the case

<img width="3442" height="1832" alt="image" src="https://github.com/user-attachments/assets/aed51403-6c81-4d13-8947-420f9447cfeb" />